CONTRIBUTING.md

```diff
@@ -24,18 +24,35 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## New Templates/Features
+
+This repo is generally not accepting contributions or feature requests for new
+templates or large chunks of new functionality unless a partner organization is
+willing to sponsor building/maintaining/supporting those templates in a formal
+agreement with Google.
+
+If you are part of an organization interested in this, please reach out to
+the Dataflow team directly using the [Google Issue Tracker](https://b.corp.google.com/issues/new?component=187168&template=1162867).
```

Comment:
is https://b.corp.google.com/issues/new?component=187168&template=1162867 accessible? I could not access it using my personal device.